src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -35,18 +33,13 @@ public AutoRegisteringObjectGraphType(params Expression<Func<TSourceType, object
         /// <summary>
         /// Returns a list of properties that should have fields created for them.
         /// </summary>
-        protected virtual IEnumerable<PropertyInfo> GetRegisteredProperties()
-        {
-            return typeof(TSourceType)
-                .GetProperties(BindingFlags.Public | BindingFlags.Instance)
-                .Where(p => AutoRegisteringHelper.IsEnabledForRegister(p.PropertyType, true));
```

Comment:
I see the `Where` portion is gone from a number of these methods.  Was that functionality moved somewhere else?  I don't actually use this class so not real familiar with this.

Reply:
This code is now simply not needed. Previously, it could filter some properties when it thought they could not be matched. Even if this happens, it means that the user must add a type mapping. It is better to tell him about this through an exception than to quietly exclude a property so that he does not waste time figuring out why some property does not exist in autogenerated graph type.

Reply:
Then we should add this to the migration notes, saying that previously, unmatched properties would be ignored, and now an exception will be thrown.

Reply:
done